### PR TITLE
DOCS-2321: Document fleet dashboard on manage fleet page

### PR DESCRIPTION
**Pull request by sguequierre**
* documents dashboard and all machines dashboard on manage machines how-to
* adds sentence documenting app locations page change on locations docs page

### CODE REVIEW

File: docs/how-tos/manage-fleet.md [hunk 1]
```diff
@@ -130,11 +130,24 @@ You can manage this in the **API Keys** section of the organization settings pag
 
 ## Monitoring and debugging
 
-Viam allows you to access each machine, check its logs, check recent changes, and roll back changes if needed.
+Viam allows you to view all machines from a dashboard and access each machine, check its logs, check recent changes, and roll back changes if needed.
 
 {{< table >}}
+{{% tablestep %}}
+**1. Monitor your fleet's synced data and part status**
```
Comment (npentrel) [inline docs/how-tos/manage-fleet.md:137]: Order is different on the screenshot
```suggestion
**1. Monitor your fleet's parts statuses and synced data**
```

File: docs/how-tos/manage-fleet.md [hunk 2]
```diff
@@ -130,11 +130,24 @@ You can manage this in the **API Keys** section of the organization settings pag
...
+
+You can monitor the amount of binary and tabular data your fleet has synced in the last 48 hours from the Viam app's **FLEET** tab's [**DASHBOARD** subtab](https://app.viam.com/fleet/dashboard).
+You can also monitor the number of parts online, offline, and awaiting setup on this dashboard.
```
Comment (npentrel) [inline docs/how-tos/manage-fleet.md:140]:
```suggestion
You can monitor the number of parts online, offline, and awaiting setup on this dashboard.
You can also monitor the amount of binary and tabular data your fleet has synced in the last 48 hours from the Viam app's **FLEET** tab's [**DASHBOARD** subtab](https://app.viam.com/fleet/dashboard).
```

File: docs/how-tos/manage-fleet.md [hunk 3]
```diff
@@ -130,11 +130,24 @@ You can manage this in the **API Keys** section of the organization settings pag
...
+
+{{<imgproc src="how-tos/manage-fleet/dashboard.png" resize="500x" declaredimensions=true alt="Fleet dashboard">}}
+
+For a table of data about each machine part, including **Location**, **Status**, **Architecture**, and more, view the **FLEET** tab's [**ALL MACHINES DASHBOARD** subtab](https://app.viam.com/fleet/machines).
```
Comment (npentrel) [inline docs/how-tos/manage-fleet.md:144]:
```suggestion
To view data at the machine part level, including **Location**, **Status**, **Architecture**, and more, go to the **FLEET** tab's [**ALL MACHINES DASHBOARD** subtab](https://app.viam.com/fleet/machines).
```

**Review by npentrel — APPROVED** [review]
Great - thank you. 2 comments then good to merge

**Review by npentrel — CHANGES_REQUESTED** [review]
Actually - I just reviewed the description of DOCS-3005 (which you moved into reivew in progress so I assume you rolled that into this change). I don't think that work is fully done. The UI changed and we definitely have to change more guidance for users than we do here. I gave the manage-fleet page a cursory review (When you create a location you can't click on the location) and there are definitely other things that still need changing. Up to you whether you want to do that in this PR but it does need doing.

File: docs/how-tos/manage-fleet.md (hunk 2 above)
Comment (sguequierre) [reply docs/how-tos/manage-fleet.md:140]: sorry, why this change? we don't explain what "this dashboard" is until the second sentence so it makes sense for it to come first

File: docs/how-tos/manage-fleet.md (hunk 3 above)
Comment (sguequierre) [reply docs/how-tos/manage-fleet.md:144]: Good suggestion!

File: docs/how-tos/manage-fleet.md [hunk 4]
```diff
@@ -72,13 +72,13 @@ If you'd like to look at an example, see [Monitor Air Quality with a Fleet of Se
 {{% tablestep link="/cloud/locations/" %}}
 **2. Create locations**
 
-1. Click **FLEET** in the upper-left corner of the page.
+1. Click **FLEET** in the upper-left corner of the page and click **LOCATIONS**.
    A new location called `First Location` is automatically generated for you.
 
-   Click on the location and use the **...** menu next to edit the location name to what it represents for your use case.
+   Use the **...** menu next to edit the location name to what it represents for your use case.
```
Comment (sguequierre) [inline docs/how-tos/manage-fleet.md:78]: did this bc it automatically takes you to it, not sure if new change

Comment (npentrel) [inline docs/how-tos/manage-fleet.md:78]: Can't parse whether the `next` means `next to` or `use next` - but it makes sense without I think
```suggestion
   Use the **...** menu to edit the location name to what it represents for your use case.
```

**Review by npentrel — APPROVED** [review]

File: docs/how-tos/manage-fleet.md (hunk 2 above)
Comment (sguequierre) [reply docs/how-tos/manage-fleet.md:140]: @npentrel just want to check and make sure you're ok with the original phrasing before I merge this

Comment (npentrel) [reply docs/how-tos/manage-fleet.md:140]: The same reason I flipped the title for this section. That's not the order the information appears on the page. 

Is there a reason why you're putting the binary and tabular data amounts first? 

```suggestion
You can monitor the number of parts online, offline, and awaiting setup from the Viam app's **FLEET** tab's [**DASHBOARD** subtab](https://app.viam.com/fleet/dashboard).
You can also monitor the amount of binary and tabular data your fleet has synced in the last 48 hours on this dashboard.
```

Comment (sguequierre) [reply docs/how-tos/manage-fleet.md:140]: Ahh yes that makes sense. I was just thinking that the data views are the main feature of the page so it made sense to say so first but happy with this suggestion. thanks!

**Pull request merged**